model/project_ref.go

```diff
@@ -142,6 +143,15 @@ type TriggerDefinition struct {
 	Alias        string `bson:"alias,omitempty" json:"alias,omitempty"`
 }
 
+type PatchTriggerDefinition struct {
+	DefinitionAlias   string `bson:"definition_alias" json:"definition_alias"`
+	ChildProject      string `bson:"child_project" json:"child_project"`
+	Status            string `bson:"status,omitempty" json:"status,omitempty"`
+	TaskRegex         string `bson:"task_regex,omitempty" json:"task_regex,omitempty"`
+	BuildVariantRegex string `bson:"buildvariant_regex,omitempty" json:"buildvariant_regex,omitempty"`
```

Comment:
to be consistent, elsewhere this is variant regex

Reply:
done